cpu/esp_common/esp-now/esp_now_netdev.c

```diff
@@ -633,7 +633,7 @@ static int _send(netdev_t *netdev, const iolist_t *iolist)
     uint8_t* _esp_now_dst = NULL;
 
     for (uint8_t i = 0; i < ESP_NOW_ADDR_LEN; i++) {
-        if (((uint8_t*)iolist->iol_base)[i] != 0xff) {
+        if ((iolist->iol_base != NULL) && ((uint8_t*)iolist->iol_base)[i] != 0xff) {
```

Comment:
Still parentheses missing around the second expression.

Reply:
Ups, I should have taken more time to fix it :sunglasses: